src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -249,4 +255,20 @@ public Section(string name, ImmutableDictionary<string, string> properties)
             public ImmutableDictionary<string, string> Properties { get; }
         }
     }
+
+    internal sealed class GlobalAnalyzerConfig
```

Comment:
Assuming that this type represents essentially a synthesized global analyzer config since there is no true one global but rather a merged version of all the individual global ones. If so consider adding a doc comment to this type that elaborates on this.